web_renderer/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "web_renderer",
+  "productName": "web_renderer",
+  "version": "1.0.0",
+  "description": "OSR Web Renderer",
+  "main": ".webpack/main",
+  "scripts": {
+    "start": "electron-forge start",
+    "package": "electron-forge package",
+    "make": "electron-forge make",
+    "publish": "electron-forge publish",
+    "lint": "eslint --ext .ts,.tsx ."
```

Comment:
`npm run lint` returns some errors
Fix them and it would be good to add those checks to the CI (also check if tsc compiles without warnings)